File: C++/Basic/factorial.cpp
#include <iostream>

int main() {
    int number;
    std::cout << "Enter a number: ";
    std::cin >> number;
    int factorial = 1;

    for (int i = 1; i <= number; ++i) {
        factorial *= i;
    }

    std::cout << "Factorial of " << number << " is " << factorial << std::endl;
    return 0;
}

File: C#/Basic/factorial.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter a number: ");
        int number = int.Parse(Console.ReadLine());
        int factorial = 1;

        for (int i = 1; i <= number; i++)
        {
            factorial *= i;
        }

        Console.WriteLine("Factorial of " + number + " is " + factorial);
    }
}
